sql/moz-fx-data-shared-prod/telemetry_derived/agg_active_users_v1/query.sql

```diff
@@ -0,0 +1,57 @@
+-- Aggregated clients data including active users, new profiles and search metrics
```

Comment:
Move this to the `metadata.yaml` description:
https://github.com/mozilla/bigquery-etl/pull/2952/files#diff-2e37bcde22da96b36b063787a0b603e21093ec0a23a11f2657744b0e530a9f99R2

This is more useful there as when looking at the table in BQ UI it will show that description.

Reply:
I find it useful to have an overall description of the query, it gives quick idea of what it does. A more detailed description is in the metadata file too.